2.0/model-index.json

```diff
@@ -567,6 +568,14 @@
       "glTF": "TwoSidedPlane.gltf"
     }
   },
+  {
+    "name": "UnicodeTest",
+    "screenshot": "screenshot/screenshot.png",
+    "variants": {
+      "glTF": "model-\u2764\u267b-model.gltf",
+      "glTF-Binary": "model-\u2764\u267b-model.glb"
```

Comment:
Did this thing choke our Python script here?  Or was this run before a rename?

Reply:
No, Python's `json.dump` writes pure ASCII output by default; any non-ASCII characters are written escaped. If you want to change it, pass `ensure_ascii=False` [here](https://github.com/KhronosGroup/glTF-Sample-Models/blob/0bc2ad7535aba4e16fb1afba1f85fd827bbbea6d/generate-index.py#L37).

Reply:
What format would you prefer? Escaped unicode characters like now, or the original characters?

Reply:
Btw ensure_ascii is not enough, the final code should look something like this:
```python
with codecs.open("model-index.json", "w", encoding="utf-8") as f:
    json.dump(index, f, indent=2, sort_keys=True, ensure_ascii=False)
    f.write("\n")
```

Reply:
Apologies for the noise here.  Escaped is totally fine.  Somehow this looked broken to me last night, but it's correct.